src/fs/transaction.rs

```diff
@@ -19,6 +19,11 @@ use super::mode::{as_file_kind, as_file_perm, make_mode};
 use super::reply::{DirItem, StatFs};
 use super::tikv_fs::{DIR_PARENT, DIR_SELF};
 
+// default from tikv-rust-client: no_jitter_backoff(base_delay_ms: 2, max_delay_ms: 500, max_attempts: 10)
+pub const DEFAULT_REGION_BACKOFF: Backoff = Backoff::no_jitter_backoff(300, 1000, 100);
+// default from tikv-rust-client: no_jitter_backoff(base_delay_ms: 2, max_delay_ms: 500, max_attempts: 10)
```

Comment:
Please keep the comments consist with the real value.

Reply:
It's intendend that they differ. I put them as reference to show what the defaults are.
Anyway. Meanwhile I did some further testing and it seems that I have some issues with the WIFI. When I do the same test with a cable ethernet connection to the cluster, results are much better. So I think its probably better to implement a command line option that switches two predefined set of values. E.g. one for "Low Latency Networks" and one for "Wireless/Mobile" connections. 
What do you think?

Reply:
In my opinion, I prefer to setting a low `base_delay_ms` as the default value, and left a command option and description to hint users to use a higher `base_delay_ms` in unstable network. The `max_attempts ` can be a constant value, and the `max_delay_ms` may be `N * base_delay_ms`. 

Providing two options to be suit for different network environments is a black-box.